cypress/integration/analyticsFlows/analyticsNavigation.spec.js

```diff
@@ -43,4 +43,27 @@ describe('Analytics navigation', () => {
       .findByRole('button', { name: 'Infinity' })
       .should('have.attr', 'aria-current', 'page');
   });
+
+  it('should hide organizations menu', () => {
+    cy.findByRole('navigation', { name: 'Organizations menu' }).should(
+      'not.exist',
+    );
+  });
+});
+
+describe('when user is admin of an organization', () => {
+  before(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.visit('/dashboard/analytics');
+      });
+    });
+  });
+
+  it('should show organizations menu', () => {
+    cy.findByRole('navigation', { name: 'Organizations menu' }).should('exist');
+  });
```

Comment:
small refactor (blocking) - I think we should also assert that we're taken to the right organisation dashboard:

- click on the link for the org dashboard
- check that the heading is 'Analytics Dashboard for {company name}'

To enable that I think you'll need to specify a fixed name for the organisation in `seeds_e2e.rb`, replacing the Faker call 👍

Reply:
Great suggestion, will add a test for this!